trunk/research/console/en_index.html

```diff
@@ -20,7 +20,7 @@
     <script type="text/javascript" src="js/srs.console.js"></script>
 </head>
 <body ng-controller="CSCMain">
-<img src='//ossrs.net/gif/v1/sls.gif?site=ossrs.net&path=/console/enindex'/>
+<img style="width: 1px; height: 1px;" src='//ossrs.net/gif/v1/sls.gif?site=ossrs.net&path=/console/enindex'/>
```

Comment:
Why not change to:

```css
width: 0px; height: 0px;
```

Reply:
`with: 0px; height: 0px;` also works, but chrome will render this `img` tag in `1px` size by default, so I did this to keep consistency with chrome.

Reply:
change to `width: 0px; height: 0px`, and confirmed the `gif` request can still send: `http://ossrs.net/gif/v1/sls.gif?site=ossrs.net&path=/player/srsplayer`.